src/pages/Home/index.tsx

```diff
@@ -51,6 +51,14 @@ const Home = () => {
         icon="waving.svg"

```

Comment:
- **Pre-condition check**  
  The `AboutContent` object is not checked for null or undefined before being used. This could lead to a runtime error if `AboutContent` is null or undefined.  
- **Runtime error check**  
  The `ContentBlock` component is not checked for null or undefined before being used. This could lead to a runtime error if `ContentBlock` is null or undefined.